src/components/builds/BuildDetails.tsx

```diff
@@ -168,6 +170,10 @@ class BuildDetails extends React.Component<Props> {
           <CardContent>
             <div className={classes.wrapper}>
               <RepositoryNameChip className={classes.chip} repository={build.repository} />
+              <BuildBranchNameChip className={classes.chip} build={build} />
```

Comment:
This information is available under the `changeMessageTitle`. Will it be enough?

Reply:
Do you mean from a code perspective or visual perspective?

Reply:
Both. Also from UX perspective links below open external links of the brnach/PR and the commit in GH interface where chips always open a Cirrus page.

Reply:
@AlexWayfer wanted this, plus isn't it just better to have these chips for consistency?

Reply:
After taking some time to rethink this, I do still believe this change is for the better.  Why not have Cirrus links there, along with GitHub ones?

Reply:
`BuildChangeChip` will lead to the same page. We can add just the `BuildBranchNameChip`. What do you think?

Reply:
That works fine for me.  I'll change it.